src/manage_config_discovery.c

```diff
@@ -941,7 +1480,8 @@ make_config_discovery (char *const uuid, char *const selector_name)
   sql ("INSERT INTO config_preferences (config, type, name, value)"
        " VALUES ((SELECT id FROM configs WHERE uuid = '%s'),"
        "         'PLUGINS_PREFS',"
-       "         '" OID_PING_HOST ":checkbox:Mark unrechable Hosts as dead (not scanning)',"
+       "         '" OID_PING_HOST
```

Comment:
like the other "macro inside a string" cases, this would be better if we used a %i for OID_PING_HOST.